### PR TITLE
Sorting, and stuff

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher , @c0mster , @birdage , @dlangston Please review!

### CODE REVIEW

File: ooiui/static/js/partials/AssetEventsTable.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
                                 data-target="#cal-<%= row.eventId %>">
                                     <td><%= row.eventId  %></td>
                                     <td><%= row.class %></td>
-                                    <<td><%= new Date(row.startDate).toUTCString() %></td>
```
Comment (birdage) [inline ooiui/static/js/partials/AssetEventsTable.html]: @FBRTMaka nice catch.

File: ooiui/static/js/views/science/StreamTableView.js [hunk 2]
```diff
@@ -15,85 +15,98 @@
...
+        this.render();
+    },
+    onDownload: function(event) {
+        event.stopPropagation();
+        event.preventDefault();
+        var option = $(event.currentTarget).attr("download-type");
+        ooi.trigger('StreamTableItemView:onClick', {model: this.model, selection: option});
+    },
+    focus: function() {
+        console.log("Trying to focus");
```
Comment (birdage) [inline ooiui/static/js/views/science/StreamTableView.js]: @FBRTMaka we should remove this down the line

File: ooiui/templates/asset_management/assetslist.html [hunk 3]
```diff
@@ -138,6 +138,12 @@
     this.listenTo(this, "login:success", this.onLogin);
     this.listenTo(this, "login:logout", this.onLogout);
     // TOC Controls
+    this.listenTo(this, 'toc:selectArray', function(model) {
+        var searchFor = model.get('array_name').substr(0,14);
+        $('#search').val(searchFor);
```
Comment (birdage) [inline ooiui/templates/asset_management/assetslist.html]: @FBRTMaka no `this.$el`?

File: ooiui/templates/science/plotting.html [hunk 4]
```diff
@@ -450,20 +495,20 @@
...
+      self.views.annotationTableView.render();
     });
 
-    //  
+    //
     this.listenTo(this.collections.plotEventList, 'reset', function(collection) {
       self.views.plotEventListView.collection = collection;
       $.when(self.views.plotEventListView.render()).done(function() {
-        $('#eventsTable').fixedHeaderTable({ altClass: 'odd', footer: true, fixedColumns: false, });  
+        $('#eventsTable').fixedHeaderTable({ altClass: 'odd', footer: true, fixedColumns: false, });
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: @FBRTMaka `,` at the end of the line wont break anything?

File: ooiui/templates/science/streams.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
 {% block head %}
 
   <link href="/css/compiled/index.css" rel="stylesheet" type="text/css" />
-  <link href="/css/compiled/index.css" rel="stylesheet" type="text/css" />
+  <link href="/css/common/streams.css" rel="stylesheet" type="text/css" />
```
Comment (birdage) [inline ooiui/templates/science/streams.html]: @FBRTMaka :+1:

File: ooiui/templates/science/plotting.html (hunk 4 above)
Comment (FBRTMaka) [inline ooiui/templates/science/plotting.html]: @birdage you mean the removal of the space? nope! lol

**Comment by FBRTMaka** [comment]
@DanielJMaher , didn't realize this PR has been open for this long.  Added the update to the TOC to the end of this.  Overall this should probably get merged at some point! haha

**Comment by DanielJMaher** [comment]
@birdage Please review and merge

**Comment by FBRTMaka** [comment]
@birdage yay!

**Pull request merged**